cmd/tenant/main.go

```diff
@@ -29,6 +37,33 @@ func main() {
 			Action:   serve,
 			Flags:    []cli.Flag{},
 		},
+		{
+			Name:     "db:list",
+			Usage:    "list all keys in a tenant namespace",
```

Comment:
We might want to include this in main - I found it pretty useful for debugging, especially since the IDs are ordered.

Reply:
sounds good!